### PR TITLE
Update: bring feature flags documentation up to date

### PR DESCRIPTION
**Pull request by marcvberg**
_Issue #, if available:_

_Description of changes:_
Brings the feature flags documentation up to date on missing feature flags

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by josefaidt — APPROVED** [review]

**Review by edwardfoyle — APPROVED** [review]

File: src/components/FeatureFlags/feature-flags.json [hunk 1]
```diff
@@ -130,6 +150,87 @@
...
+      "suppressSchemaMigrationPrompt": {
+        "description": "Prevent the CLI from prompting for schema migration",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "6.4.0",
+        "deprecationDate": "November 10th 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "V1 GraphQL Transformer projects will not be prompted to migrate to V2",
```
[the remark this answers is not in the document]
Comment (marcvberg) [reply src/components/FeatureFlags/feature-flags.json]: Fixed

File: src/components/FeatureFlags/feature-flags.json [hunk 2]
```diff
@@ -224,6 +325,46 @@
...
+          {
+            "value": "false",
+            "description": "Continue to use usernameAttributes with Cognito",
+            "defaultNewProject": true,
+            "defaultExistingProject": true
+          }
+        ]
+      },
+      "useEnabledMfas": {
+        "description": "Allows the use of only TOTP if only phone_number is configured, if another attribte is added SMS MFA is enabled",
```
[the remark this answers is not in the document]
Comment (marcvberg) [reply src/components/FeatureFlags/feature-flags.json]: Fixed

File: src/components/FeatureFlags/feature-flags.json [hunk 3]
```diff
@@ -130,6 +150,87 @@
...
+      },
+      "transformerVersion": {
+        "description": "Sets which version of the GraphQL Transformer will be used",
+        "type": "Feature",
+        "valueType": "Number",
+        "versionAdded": "6.4.0",
+        "values": [
+          {
+            "value": "1",
+            "description": "The original V1 GraphQL Transformer will be used",
```
[the remark this answers is not in the document]
Comment (marcvberg) [reply src/components/FeatureFlags/feature-flags.json]: Fixed

File: src/components/FeatureFlags/feature-flags.json [hunk 4]
```diff
@@ -48,6 +48,26 @@
           }
         ]
       },
+      "improvePluralization": {
+        "description": "Use the pluralize npm library instead of only appending an \"s\" to achieve pluralization.",
```
[the remark this answers is not in the document]
Comment (marcvberg) [reply src/components/FeatureFlags/feature-flags.json]: Fixed

**Review by cjihrig — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Pull request merged**